jitbuilder/build/rules/common.mk

```diff
@@ -123,6 +123,42 @@ $(RELEASE_INCLUDE)/$(JIT_OMR_DIRTY_DIR)/ilgen/OMRMethodBuilder.hpp: $(FIXED_SRCB
 $(RELEASE_INCLUDE)/$(JIT_OMR_DIRTY_DIR)/ilgen/MethodBuilder.hpp: $(FIXED_SRCBASE)/$(JIT_OMR_DIRTY_DIR)/ilgen/MethodBuilder.hpp $(RELEASE_INCLUDE)/$(JIT_OMR_DIRTY_DIR)/ilgen
 	cp $< $@ || cp $< $@
 
+$(RELEASE_INCLUDE)/$(JIT_OMR_DIRTY_DIR)/ilgen/JitbuilderRecorder.hpp: $(FIXED_SRCBASE)/$(JIT_OMR_DIRTY_DIR)/ilgen/JitbuilderRecorder.hpp $(RELEASE_INCLUDE)/$(JIT_OMR_DIRTY_DIR)/ilgen
```

Comment:
I believe those files would have been removed from the Recorder PR because they are not actually being used. They should probably be removed from this PR as well.

Reply:
I believe there is still one (x86 even) platform that builds with autoconf, so the changes in these files are relevant, but I don't think the `*Recorder` files should be in the PR for the `Replay` classes :) (or should at the very least be in a separate commit by themselves).

Reply:
The `*Recorder` part of this file is missing in the original PR. I'll remove those from this PR.